packages/client/src/knock.ts

```diff
@@ -130,8 +130,8 @@ class Knock {
     }
   }
 
-  log(message: string) {
-    if (this.logLevel === "debug") {
+  log(message: string, force = false) {
```

Comment:
Any reason to use this vs. setting the log level?

Reply:
I guess i could have set log-level and then set it back – just felt like it was cleaner to have an override